runtime/doc/api.txt

```diff
@@ -1284,7 +1284,7 @@ nvim_set_current_dir({dir})                           *nvim_set_current_dir()*
       • {dir}  Directory path
 
 nvim_set_current_line({line})                        *nvim_set_current_line()*
-    Sets the current line.
+    Changes the text on the current line.
```

Comment:
```suggestion
    Sets the current line text.
```

Reply:
nvim_set_current_line is completely different from nvim_set_current_buf -- one changes content and the other navigates. I think the proposed text makes that distinction less clear.

What about:

> Sets the text on the current line.